CentralComputing/ADCManager.cpp

```diff
@@ -105,4 +110,31 @@ void ADCManager::initialize_sensor_error_configs() {
 }
 
 void ADCManager::check_for_sensor_error(const std::shared_ptr<ADCData> & check_data) {
+  //Just hardcoding and using difference for accelerometers for now
```

Comment:
Depending on what exactly the ADC mapping is, the indicies might need to be changed around. Otherwise this looks great!